meson.build

```diff
@@ -278,6 +278,11 @@ if lzma.found()
   conf.set('HAVE_LZMA', '1')
 endif
 
+cbor = dependency('libcbor', required: get_option('cbor'))
```

Comment:
what version are you starting with on your side?  0.8?  I'd rather anchor it somewhere if we can.

Reply:
0.7.0 here, good idea.

Reply:
Fixed.